lib/utils/addEntries.js

```diff
@@ -20,7 +20,7 @@ function addEntries(config, options, server) {
     const sockPath = options.sockPath ? `&sockPath=${options.sockPath}` : '';
     const sockPort = options.sockPort ? `&sockPort=${options.sockPort}` : '';
     const clientEntry = `${require.resolve(
-      '../../client/'
+      '../../client/default/'
```

Comment:
Breaking change, some packages can use this directly, so changing path can break these apps